content/docs/pulumi-cloud/deployments/api.md

```diff
@@ -380,6 +381,24 @@ The GitHub block describes settings for Pulumi Deployments' GitHub integration.
 }
 ```
 
+### CacheOptions
```

Comment:
We have inconsistent casing and spacing on various terms here. "CacheOptions" vs "Cache Options" vs `cacheOptions` vs "cache options"...

For the H3 use sentence casing: "Cache options"
In the paragraph text, don't capitalize (it's not a proper noun): "cache options" or use the specific symbol `cacheOptions`, depending on wether you're referring to the concept of cache options or the JSON syntax for it. 

In paragraph text "during Deployments" should be "during deployments" (we're talking about the action of deploying, not the product name). Usage of "Dependency Caching" here should also be lower case.